test/test_classifier.rb

```diff
@@ -50,7 +50,7 @@ def test_classify_ambiguous_languages
       next unless languages.length > 1
 
       results = Classifier.classify(Samples.cache, File.read(sample[:path]), languages)
-      assert_equal language.name, results.first[0], "#{sample[:path]}\n#{results.inspect}"
+      assert results.map{|key,value| key}.include?(language.name), "#{sample[:path]}\n#{results.inspect}"
```

Comment:
I'm wondering whether it was written that way on purpose... I guess, because we trained the classifier in part with the sample we are now trying to classify, it should be correctly detected and have the strongest score assigned (thus being the first in the list).